src/Lotto.js

```diff
@@ -10,6 +10,17 @@ class Lotto {
     if (numbers.length !== 6) {
       throw new Error("[ERROR] 로또 번호는 6개여야 합니다.");
     }
+
+    const set = new Set(numbers);
+    if (numbers.length !== set.size) {
+      throw new Error("[ERROR] 로또 번호에 중복된 숫자가 있습니다.");
+    }
+
+    for (let i = 0; i < numbers.length; i++) {
+      if (numbers[i] < 1 || numbers[i] > 45) {
+        throw new Error("[ERROR] 로또 번호가 1부터 45 사이의 숫자가 아닙니다");
+      }
+    }
```

Comment:
```suggestion
    for (const number of numbers) {
      if (number < 1 || 45 < number) {
        throw new Error("[ERROR] 로또 번호가 1부터 45 사이의 숫자가 아닙니다");
      }
    }
```
`for..of` 를 사용하면 불필요한 `i` 변수를 사용하지 않아 더 깔끔해집니다!

`if (number < 1 || 45 < number)` 이렇게 오름차순 모양으로 만들면 자연스럽게 읽을 수 있을 것 같습니다!